packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -67,9 +67,9 @@ export async function fetchSnapshot(
 
     if (fetchFullSnapshot) {
         if (versionId !== "latest") {
-            queryParams = { channels: 1, blobs: 2 };
+            queryParams = { blobs: 2 };
         } else {
-            queryParams = { deltas: 1, channels: 1, blobs: 2 };
+            queryParams = { deltas: 1, blobs: 2 };
```

Comment:
I'm a bit confused on the flow. 
1. This is only used through getSnapshotTree() call, and fetchFullSnapshot is always true (in production code). And only if we do not have tree in cache. I'd assume we should always download just a tree, i.e. fetchFullSnapshot should be always false, no?
2. It feels like the most important stuff we need is channels, does dropping them means - we use default (they are included), or not?

Reply:
1.) I am not aware why fetchFullSnapshot was added and why it is set to true in the production. Just wanted to keep the behavior as is. However, this does not do anything as by default they supply both blobs and deltas. So I am going to remove them.
2.) I confirmed that Channels is not even processed by the server. Only deltas and blobs. So, they always supply trees by default. That is why I am removing them here.